homeassistant/components/awair/__init__.py

```diff
@@ -19,35 +18,22 @@
 
 async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
     """Set up Awair integration from a config entry."""
-    session = async_get_clientsession(hass)
-
     coordinator: AwairDataUpdateCoordinator
 
     if CONF_HOST in config_entry.data:
-        coordinator = AwairLocalDataUpdateCoordinator(hass, config_entry, session)
-        config_entry.async_on_unload(
```

Comment:
This has not moved, but has been removed. On purpose?

Reply:
I was checking why this was there in the first place. And it seems to be reloading whenever a entry title change because it's the device name. Wouldn't it make sense to take this up globally?